installations/Podfile

```diff
@@ -0,0 +1,13 @@
+platform :ios, '9.0'
+
+# Replace both pods by pod 'Firebase/Installations' once FirebaseInstallations is public.
+pod 'FirebaseInstallations', :git => 'https://github.com/firebase/firebase-ios-sdk.git', :branch => 'master'
+pod 'Firebase/CoreOnly'
+
+target 'InstallationsExample' do
+  use_frameworks!
```

Comment:
nit: can push this line to above to avoid duplication

Reply:
done